pinot-servicemanager/docker-bin/install-schema

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+#
+# The schemas directory is a HEALTHCHECK barrier, deleted when this completes.
+set -eu
+
+IP="$(hostname -i || echo '127.0.0.1')"
+
+timeout=60
```

Comment:
Hmm!! In some cases, can we expose this as ENV?

Reply:
sure thing. INSTALL_SCHEMA_TIMEOUT? it would be a little imprecise as this is only the wait time. and FWIW for me it is <30s. Another way is to make it a lot longer so that we don't need a variable. ex 180

Reply:
I made this excessively high. 6x my norms